mcs/class/Mono.Posix/Mono.Unix.Native/Syscall.cs

```diff
@@ -727,6 +727,194 @@ public enum MremapFlags : ulong {
 		MREMAP_MAYMOVE = 0x1,
 	}
 
+	[Map]
+	[CLSCompliant (false)]
+	public enum SockType : int {
```

Comment:
Isn't the point of C# namespaces to not care about naming collisions? If some codebase is going to use both Socket classes, one can use an alias via "using UnixSocket = Mono.Posix.Socket;"